tools/devops/automation/build-pipeline.yml

```diff
@@ -92,6 +94,7 @@ parameters:
       stageName: 'mac_10_13',
       displayName: 'Mac High Sierra (10.13)',
       macPool: 'Hosted Mac Internal',
```

Comment:
You can use the `internal-macos-10.13` image in place of the `Hosted Mac Internal` pool